server/test/integration/com/thoughtworks/go/config/GoConfigMigrationIntegrationTest.java

```diff
@@ -1502,6 +1502,20 @@ public void shouldMigrateLdapToAuthConfigAsPartOfMigration91() throws Exception
     }
 
     @Test
+    public void shouldAddIdOnConfigRepoAsPartOfMigration94() throws Exception {
+        String configXml = "<cruise schemaVersion='93'>" +
+                "<config-repos>\n" +
+                "   <config-repo plugin=\"json.config.plugin\">\n" +
```

Comment:
Its not. I am not saying write a test. Just asking you to confirm that there's no unexpected behavior when you have multiple config repos